bitnami/zookeeper/templates/secrets.yaml

```diff
@@ -32,15 +33,11 @@ metadata:
   {{- end }}
 type: Opaque
 data:
-  {{- if or .Values.tls.client.keystorePassword .Values.tls.client.autoGenerated }}
-  keystore-password: {{ (.Values.tls.client.keystorePassword | default (randAlphaNum 10)) | b64enc | quote }}
-  {{- end }}
-  {{- if or .Values.tls.client.truststorePassword .Values.tls.client.autoGenerated }}
-  truststore-password: {{ (.Values.tls.client.truststorePassword | default (randAlphaNum 10))| b64enc | quote }}
-  {{- end }}
+  keystore-password: {{ default (randAlphaNum 10) .Values.tls.client.keystorePassword | b64enc | quote }}
+  truststore-password: {{ default (randAlphaNum 10) .Values.tls.client.truststorePassword | b64enc | quote }}
```

Comment:
Why remove the conditionals? It is not common and I think you need to do it intentionally, but as far as I know, Truststore and Keystore can be created without a password.

Reply:
The 1st conditional didn't make much sense if random password can be created when it's not provided.
On the other hand, the second conditional didn't make sense, it's intended to auto-generate the TLS secrets instead of the associated passwords.